en-zu/jw300-baseline/config.yaml

```diff
@@ -0,0 +1,85 @@
+
+name: "enzu4000baseline_transformer"
+
+data:
+    src: "en"
+    trg: "zu"
+    train: "/content/drive/My Drive/masakhane/en-zu-baseline/train.bpe"
+    dev:   "/content/drive/My Drive/masakhane/en-zu-baseline/dev.bpe"
+    test:  "/content/drive/My Drive/masakhane/en-zu-baseline/test.bpe"
+    level: "bpe"
+    lowercase: False
+    max_sent_length: 100
+    src_vocab: "/content/drive/My Drive/masakhane/en-zu-baseline/vocab.txt"
+    trg_vocab: "/content/drive/My Drive/masakhane/en-zu-baseline/vocab.txt"
+
+testing:
+    beam_size: 5
+    alpha: 1.0
+
+training:
+    load_model: "/content/drive/My Drive/masakhane/en-zu-baseline/pretrained/enzu4000baseline/101000.ckpt" # if uncommented, load a pre-trained model from this checkpoint
```

Comment:
can we make these relative? 
@juliakreutzer What's your preference here?

Reply:
I think it's fine :) these models will have to be downloaded from individual links anyway... So at the moment one has to adapt the `load_model` and vocabulary paths manually in order to use pretrained models.